web_client/src/App.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component, Fragment } from 'react';
+import { Route } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { ConnectedRouter } from 'connected-react-router';
```

Comment:
how do you like it?

Reply:
I like it. Kind of weird though. Initially there was [react-router-redux](https://github.com/reactjs/react-router-redux) which has 7.6k stars. That was moved [into react-router](https://github.com/ReactTraining/react-router/tree/master/packages/react-router-redux) to be supported alongside react router.

Now that is deprecated and they recommend using [connected-react-router](https://github.com/supasate/connected-react-router) which is 2 years old and only has 780 stars

Reply:
Basically react-router-redux just specifically said to use connected-react-router, without much explanation why. (Thoug hi guess the explanation is it's not being maintained)